### PR TITLE
Feat/build version

### LINKED ISSUE #676: Add build version to meta
The idea is to mark build with commit + branch + timestamp. So we can be sure we're having the correct version on remote server.

- [x] Should be off by default (you can check VUE_APP_INCLUDE_DEMOS flag as example).
- [ ] Should go in meta.
- [x] Feature should be documented (in vuestic-admin, for vuestic-ui it's dev only).
 **Not applyed to vuestic-ui.**

- [x] After you're done, please create an issue in vuestic-ui to implement this. Optionally, you can continue with implementation by yourself.

Some code for reference (please don't just copy):

```js
  // vue.config.js
  chainWebpack: config => {
    config.plugin('define').tap(definitions => {
      definitions[0] = Object.assign(definitions[0], {
        'VERSION_INFO': JSON.stringify(generateFooter())
      })
      return definitions
    })
  },
```

```
// generate-footer.js
function generateFooter () {
  const commit = getLastCommitHash()
  const timestamp = new Date().toUTCString()
  return `Copyright 2011-2019 Vuestic, Build: ${commit}, ${timestamp}`
}

function getLastCommitHash () {
  const hash = require('child_process').execSync('git rev-parse HEAD')
  return hash.slice(0, 6)
}

module.exports = {
  generateFooter
}
```

### PR DESCRIPTION
**Pull request by Kreezag**
## Close #676 
Add footer to app
Add copyright text and build information to footer

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -14,18 +14,18 @@
       }
```
Comment (asvae) [inline package-lock.json]: package-lock is updated only before release. No point to update it for every PR.

File: docs/pre-production.md [hunk 2]
```diff
@@ -36,3 +36,13 @@ You can enable demos in build by:
...
+
+
+## Build Version
+
+You can enable build version, commit hash and timestamp by build in main file:
+```
+VUE_APP_BUILD_VERSION
+```
+Build version are excluded in staging build by default. 
+To include version to build should you most add `VUE_APP_BUILD_VERSION=true` in `.env` file.
```
Comment (asvae) [inline docs/pre-production.md]: Probably remove this line.

File: vue.config.js [hunk 3]
```diff
@@ -26,19 +36,27 @@ module.exports = {
...
     },
-    plugins: [new StylelintPlugin({
-      files: ['src/**/*.{vue,htm,html,css,sss,less,scss}'],
-    })],
+    plugins: [
+      new StylelintPlugin({
+        files: ['src/**/*.{vue,htm,html,css,sss,less,scss}'],
+      }),
+      new webpack.DefinePlugin({
+        __VUE_APP_BUILD_VERSION__: process.env === 'VUE_APP_BUILD_VERSION',
```
Comment (asvae) [inline vue.config.js]: What this is even?

File: vue.config.js [hunk 4]
```diff
@@ -26,19 +36,27 @@ module.exports = {
...
+        __VERSION__: JSON.stringify(version),
```
Comment (asvae) [inline vue.config.js]: You can use VERSION. All caps means global constant.

File: package-lock.json (hunk 1 above)
Comment (Kreezag) [reply package-lock.json]: Sorry, I tried to add new dependencies while performing tasks

**Pull request merged**